exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -22,6 +23,11 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/traceutil"
 )
 
+var allowRandomTimestamp = featuregate.GlobalRegistry().MustRegister(
```

Comment:
nit: maybe change to skipTimestampValidation, we are not allowing random timestamp, we are allowing any random number with same length, it doesn't need to be a valid timestamp like 16xxxxx, 0000...01 also work.

Reply:
Added suggestion